### DISCUSSION TITLE
Help me get the Github Student Pack

### DISCUSSION THREAD

**Discussion by hepwelnaj** [community/community#20115] [GitHub Education]
Good afternoon! I am a student of Nosov Magnitogorsk State Technical University (Magnitogorskiy Gosudarstvennyy Tekhnicheskiy Universitet Im. G.i. Nosova) [magtu.ru](https://www.magtu.ru/) I have an email address with the domain of this organization (@magtu.ru). But I can't verify my status as a student because the github system does not allow my photo with a student ID. Please help me to get the Github student benefits. Attached is a scan of my student card. Thanks

**Comment by D7EAD** [comment]
Hi!

If you're having trouble acquiring the Github Student Pack, kindly reach out to Github Education's Support Engineers to help resolve this issue. You can reach them [here](https://support.github.com/contact/education).

**Comment by AyeshGK** [comment]
You can try student card, which have your photo.

**Comment by hepwelnaj** [comment]


**Reply by MeesJ** [reply]
You can try contacting GitHub Education support [here](https://support.github.com/contact/education).

**Comment by ritzwasfound** [comment]
Try submitting a dated ID that shows until what year the ID is valid, or you can try submitting a class schedule for the current semester. I did the latter and got the GitHub Student Pack in less than a week. Good luck on re-applying 🥨.

**Comment by Sendbaster** [comment]


**Reply by MeesJ** [reply]
Have you contacted GitHub Education [support](https://support.github.com/contact/education) already?

**Comment by hepwelnaj** [comment]


**Reply by Sendbaster** [reply]
Through support? Or Request?

**Comment by SagarDNG** [comment]
> Good afternoon! I am a student of Nosov Magnitogorsk State Technical University (Magnitogorskiy Gosudarstvennyy Tekhnicheskiy Universitet Im. G.i. Nosova) [magtu.ru](https://www.magtu.ru/) I have an email address with the domain of this organization (@magtu.ru). But I can't verify my status as a student because the github system does not allow my photo with a student ID. Please help me to get the Github student benefits. Attached is a scan of my student card. Thanks

Hey,
Did you get it?
